### ISSUE TITLE
Add jQuery UI Theme Switcher to the demo page

### ISSUE THREAD

**Issue by arshaw** [fullcalendar/fullcalendar#1436]
Originally reported on Google Code with ID 1169

```
Might be useful to add the dynamic theme switcher to the query ui theme demo page.

In the header add:
<link href="//ajax.googleapis.com/ajax/libs/jqueryui/1.8.16/themes/ui-lightness/jquery-ui.css"
rel="stylesheet" type="text/css"/>

...add the Switch placement code:
<div id="switcher"></div>

... and then add this to your JS
$('#switcher').themeswitcher();

That's it (other than you can add some positioning CSS for the switcher div)
```

Reported by `StlDoug` on 2011-11-06 19:49:13

Imported with **1** stars.

**Comment by arshaw** [comment]
this is queued for next release. closing. there will be a demo html file that lets you preview multiple jquery ui themes as well as Bootstrap themes. see https://github.com/fullcalendar/fullcalendar/pull/3566

**Comment by arshaw** [comment]
Released in v3.5.0 ([blog post](https://fullcalendar.io/blog/2017/08/bootstrap-support-and-future-developments/))

https://fullcalendar.io/js/fullcalendar-3.5.0/demos/themes.html
